ui/components/MesheryMeshInterface/PatternService/index.js

```diff
@@ -64,6 +64,22 @@ function PatternService({ formData, jsonSchema, onChange, type, onSubmit, onDele
       });
     return sortedProperties;
   };
+  // add p_type="array" to objects in array
+  const addPTypeToArray = (properties) => {
```

Comment:
what is p_type?

Reply:
Its so good that you added the comment, just a little more info would be great.

Reply:
p_type is the parent type I am providing to the objects in array, so that with it I alter the behaviour of objects in array.

Reply:
If array have object then every object inside it carry on the p_type="array" .

Reply:
@harkiratsm please add this detail in comment in the code.

Reply:
@leecalcote updated the comment. :+1: